packages/react-components/src/templates/Layout.tsx

```diff
@@ -175,6 +180,12 @@ const Layout: React.FC<LayoutProps> = ({
   useEffect(() => {
     setMenuShown(false);
   }, [location]);
+  useEffect(() => {
+    if (location?.pathname !== prevLocation?.pathname) {
+      const main = document.querySelector('main');
```

Comment:
![nope nope nope](https://media2.giphy.com/media/yMaLDA976YtUs/200.gif)

use a ref :D

Reply:
Whoops, I did mean to convert it to a ref.